device/cuda/include/traccc/cuda/clusterization/clusterization_algorithm.hpp

```diff
@@ -67,12 +78,31 @@ class clusterization_algorithm
     /// @param det_descr The detector description
     /// @return a measurement collection (buffer)
     ///
-    output_type operator()(
+    /// @{
+    measurement_collection_types::buffer operator()(
         const edm::silicon_cell_collection::const_view& cells,
         const silicon_detector_description::const_view& det_descr)
         const override;
 
+    measurement_collection_types::buffer operator()(
+        const edm::silicon_cell_collection::const_view& cells,
+        const silicon_detector_description::const_view& det_descr,
+        device::clustering_discard_disjoint_set&&) const override;
```

Comment:
Do we need this version of the function?

Reply:
This makes it slightly easier to use the function with `std::conditional` if that is ever desired.